unlock-app/src/components/interface/locks/Settings/forms/VerifierForm.tsx

```diff
@@ -104,13 +112,30 @@ export const VerifierForm = ({
     return response.data.results || []
   }
 
-  const addVerifier = async (address: string) => {
+  const addVerifier = async ({
+    address,
+    name,
+  }: {
+    address: string
+    name?: string
+  }) => {
     const resolvedAddress = await getAddressForName(address)
-    const response = await storage.createVerifier(
-      network,
-      lockAddress,
-      resolvedAddress
-    )
+    let response
+    if (name) {
+      response = await storage.createVerifierWithName(
```

Comment:
If you use the same API call, then you can just make the `name` optional and then this is simpler/cleaner.